torch/ao/sparsity/sparsifier/base_sparsifier.py

```diff
@@ -177,6 +178,14 @@ def convert(self):
         raise NotImplementedError('`convert` is not implemented. '
             'Please, use `torch.ao.utils.convert` instead.')
 
+    def step(self, use_path=True):
+        if not self.enable_mask_update:
+            return
+        with torch.no_grad():
+            for config in self.module_groups:
+                module = config['module']
+                self.update_mask(module, **config)
```

Comment:
NIT: Since update_mask is a member of the sparsifier object, why have it take arguments?

Reply:
Oh that's because the `update_mask` is just a logic of changing generic masks. The config specifies which layers is the update called for. I am doing it so that the user would need to write minimum amount of code.